extensions/azurePublishNew/src/node/azureResources/cosmosDb.ts

```diff
@@ -37,7 +39,7 @@ type CosmosDbProvisionResult = {
   collectionId: string;
 };
 
-type CosmosDbConfig = ResourceConfig & {
+export type CosmosDbConfig = ResourceConfig & {
```

Comment:
Consider moving this type to a common place, if used in other places.

Reply:
keeping it here because it most closely relates to the cosmosdb module